torch/_dynamo/variables/builder.py

```diff
@@ -2981,6 +2982,8 @@ def create(tx: "InstructionTranslator", value) -> VariableTracker:
             return torch._dynamo.variables.higher_order_ops.FlexAttentionBackwardHighOrderVariable(
                 value
             )
+        elif isinstance(value, types.GenericAlias):
+            return BuiltinVariable(value)
```

Comment:
Do we need to worry about the case where the GenericAlias was built from parts? For example:
```
def myfn(x, y):
    z = tuple[x, y]
```
or is it guaranteed that if we got here the parts are all constants?